tests/unit/test_shortcuts.py

```diff
@@ -0,0 +1,213 @@
+import unittest
+from unittest import mock
+
+from dbt.adapters.fabricspark.shortcuts import Shortcut, TargetName, ShortcutClient
+
+class TestShorcutClient(unittest.TestCase):
```

Comment:
yay! Thanks for adding this. One of the conventions we typically follow for naming tests is <MethodName>_<Some>_<Description>_<ExpectedOutput>().
I see that you are already following it, except the expected output part.

Reply:
sorry, i don't follow. would this mean prefacing test names with `_` or suffixing them with `_`? for the expected output part, can you give an example?

Reply:
Didn't realize the naming convention got truncated - <MethodName>_<Some>_<Description>_<**ExpectedOutput**>()

Reply:
For example, 
 test_create_shortcut_does_not_exist_succeeds
 test_create_shortcut_exists_skips_shortcut_creation